scripts/ci/update-github

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+ls -l
+
+cd dcos-nightly || exit 1
+
+if [[ $TAG_NAME ]]
```

Comment:
Have you tried if the Tag name works with our PR approach of merging PRs? I would have thought that if you have a commit which is tagged in a PR and merge it you get a merge commit as the new HEAD, but I am not fully aware if this is really true.

Reply:
Yes, the merge commit would be the HEAD of master immediately following a merge. But this isn't an issue.

If you "build now" on Jenkins from a tracked tag, it will be the HEAD (from that tags perspective). Does that make sense? Think of tags like branches on Jenkins. When triggering a build from a tag it checks out the commit of that tag.

Reply:
You are right, tried it out 👍 at least locally